test/test.js

```diff
@@ -72,23 +72,22 @@ test('strings', t=>{
         jams`{{bad key} val}`
     })
 
-    o = jams(`""`)
-    t.equal("", o) //err, quotes are being escaped for some reaosn
+    t.throws(_ => jams(""))
```

Comment:
Does anyone know why `""` is a syntax error??

Reply:
I think it's because we haven't found a way yet for javascript not to turn this into null.

Reply:
Ah ya the parser returned from EBNF doesn't accept `""` for unknown reasons...